department-of-veterans-affairs/abd-vro
# evidence-pdf GET accepts application/pdf

## What was the problem?


If accept header is specified to be "application/pdf" for  evidence-pdf GET VRO return 406.

Associated tickets or Slack threads:

- [MCP-000](https://amida.atlassian.net/browse/MCP-000)

## How does this fix it?[^1]


Allows applicatoin-pdf for evidence-pdf

## How to test this PR
- Send a evidence-pdf GET with application/pdf
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
